diff --git a/HdmiCecSink/HdmiCecSinkImplementation.cpp b/HdmiCecSink/HdmiCecSinkImplementation.cpp
index c44a7fd..4794dac 100644
--- a/HdmiCecSink/HdmiCecSinkImplementation.cpp
+++ b/HdmiCecSink/HdmiCecSinkImplementation.cpp
@@ -178,7 +178,7 @@ namespace WPEFramework
              HdmiCecSinkImplementation::_instance->addDevice(header.from.toInt());
              HdmiCecSinkImplementation::_instance->updateTextViewOn(header.from.toInt());
        }
-       void HdmiCecSinkProcessor::process (const RequestActiveSourceMessage &msg, const Header &header)
+       void HdmiCecSinkProcessor::process (const RequestActiveSource &msg, const Header &header)
        {
              printHeader(header);
              LOGINFO("Command: RequestActiveSource\n");
@@ -230,7 +230,7 @@ namespace WPEFramework
          if(!updateStatus)
          HdmiCecSinkImplementation::_instance->sendDeviceUpdateInfo(header.from.toInt());
        }
-       void HdmiCecSinkProcessor::process (const SetMenuLanguageMessage &msg, const Header &header)
+       void HdmiCecSinkProcessor::process (const SetMenuLanguage &msg, const Header &header)
        {
              printHeader(header);
              LOGINFO("Command: SetMenuLanguage Language : %s \n",msg.language.toString().c_str());
@@ -2039,7 +2039,7 @@ namespace WPEFramework
 
 
             _instance->smConnection->sendTo(LogicalAddress::BROADCAST, 
-                                        MessageEncoder().encode(RequestActiveSourceMessage()), 500);
+                                        MessageEncoder().encode(RequestActiveSource()), 500);
         }
 
         void HdmiCecSinkImplementation::setActiveSource(bool isResponse)
@@ -2103,7 +2103,7 @@ namespace WPEFramework
 
             lang = _instance->deviceList[_instance->m_logicalAddressAllocated].m_currentLanguage;
 
-            _instance->smConnection->sendTo(LogicalAddress::BROADCAST, MessageEncoder().encode(SetMenuLanguageMessage(lang)), 100);
+            _instance->smConnection->sendTo(LogicalAddress::BROADCAST, MessageEncoder().encode(SetMenuLanguage(lang)), 100);
         }
 
         void HdmiCecSinkImplementation::updateInActiveSource(const int logical_address, const InActiveSource &source )
@@ -2202,7 +2202,7 @@ namespace WPEFramework
             }
 
                         LOGINFO(" Send requestShortAudioDescriptor Message ");
-                    _instance->smConnection->sendTo(LogicalAddress::AUDIO_SYSTEM,MessageEncoder().encode(RequestShortAudioDescriptorMessage(formatid,audioFormatCode,numberofdescriptor)), 1000);
+                    _instance->smConnection->sendTo(LogicalAddress::AUDIO_SYSTEM,MessageEncoder().encode(RequestShortAudioDescriptor(formatid,audioFormatCode,numberofdescriptor)), 1000);
 
         }
 
diff --git a/HdmiCecSink/HdmiCecSinkImplementation.h b/HdmiCecSink/HdmiCecSinkImplementation.h
index d28f199..45f926b 100644
--- a/HdmiCecSink/HdmiCecSinkImplementation.h
+++ b/HdmiCecSink/HdmiCecSinkImplementation.h
@@ -69,11 +69,11 @@ namespace WPEFramework {
             void process (const InActiveSource &msg, const Header &header);
             void process (const ImageViewOn &msg, const Header &header);
             void process (const TextViewOn &msg, const Header &header);
-            void process (const RequestActiveSourceMessage &msg, const Header &header);
+            void process (const RequestActiveSource &msg, const Header &header);
             void process (const Standby &msg, const Header &header);
             void process (const GetCECVersion &msg, const Header &header);
             void process (const CECVersion &msg, const Header &header);
-            void process (const SetMenuLanguageMessage &msg, const Header &header);
+            void process (const SetMenuLanguage &msg, const Header &header);
             void process (const GiveOSDName &msg, const Header &header);
             void process (const GivePhysicalAddress &msg, const Header &header);
             void process (const GiveDeviceVendorID &msg, const Header &header);
diff --git a/HdmiCecSource/HdmiCecSourceImplementation.cpp b/HdmiCecSource/HdmiCecSourceImplementation.cpp
index cd6a2c4..2258bde 100644
--- a/HdmiCecSource/HdmiCecSourceImplementation.cpp
+++ b/HdmiCecSource/HdmiCecSourceImplementation.cpp
@@ -135,7 +135,7 @@ namespace WPEFramework
              LOGINFO("Command: TextViewOn\n");
              HdmiCecSourceImplementation::_instance->addDevice(header.from.toInt());
        }
-       void HdmiCecSourceProcessor::process (const RequestActiveSourceMessage &msg, const Header &header)
+       void HdmiCecSourceProcessor::process (const RequestActiveSource &msg, const Header &header)
        {
              printHeader(header);
              LOGINFO("Command: RequestActiveSource\n");
@@ -178,7 +178,7 @@ namespace WPEFramework
              LOGINFO("Command: CECVersion Version : %s \n",msg.version.toString().c_str());
              HdmiCecSourceImplementation::_instance->addDevice(header.from.toInt());
        }
-       void HdmiCecSourceProcessor::process (const SetMenuLanguageMessage &msg, const Header &header)
+       void HdmiCecSourceProcessor::process (const SetMenuLanguage &msg, const Header &header)
        {
              printHeader(header);
              LOGINFO("Command: SetMenuLanguage Language : %s \n",msg.language.toString().c_str());
diff --git a/HdmiCecSource/HdmiCecSourceImplementation.h b/HdmiCecSource/HdmiCecSourceImplementation.h
index e02a090..e6d1a5a 100644
--- a/HdmiCecSource/HdmiCecSourceImplementation.h
+++ b/HdmiCecSource/HdmiCecSourceImplementation.h
@@ -68,11 +68,11 @@ namespace WPEFramework {
 	        void process (const InActiveSource &msg, const Header &header);
 	        void process (const ImageViewOn &msg, const Header &header);
 	        void process (const TextViewOn &msg, const Header &header);
-	        void process (const RequestActiveSourceMessage &msg, const Header &header);
+	        void process (const RequestActiveSource &msg, const Header &header);
 	        void process (const Standby &msg, const Header &header);
 	        void process (const GetCECVersion &msg, const Header &header);
 	        void process (const CECVersion &msg, const Header &header);
-	        void process (const SetMenuLanguageMessage &msg, const Header &header);
+	        void process (const SetMenuLanguage &msg, const Header &header);
 	        void process (const GiveOSDName &msg, const Header &header);
 	        void process (const GivePhysicalAddress &msg, const Header &header);
 	        void process (const GiveDeviceVendorID &msg, const Header &header);
