diff --git a/HdmiCecSource/HdmiCecSource.cpp b/HdmiCecSource/HdmiCecSource.cpp
index e00fba1..f2b260e 100644
--- a/HdmiCecSource/HdmiCecSource.cpp
+++ b/HdmiCecSource/HdmiCecSource.cpp
@@ -30,7 +30,7 @@
 
 using namespace WPEFramework;
 
-
+extern "C" void __gcov_exit();
 namespace WPEFramework
 {
     namespace {
@@ -102,7 +102,7 @@ namespace WPEFramework
         void HdmiCecSource::Deinitialize(PluginHost::IShell* service)
         {
            LOGWARN("Deinitialize HdmiCecSource plugin \n");
-
+            __gcov_exit();
            ASSERT(nullptr != service);
            
 
