diff --git a/dsmgr/Makefile b/dsmgr/Makefile
index 93ce192..2a39a64 100755
--- a/dsmgr/Makefile
+++ b/dsmgr/Makefile
@@ -62,7 +62,7 @@ LDFLAGS += -L$(SDK_FSROOT)/usr/local/lib -lIARMBus
 LDFLAGS += -L$(INSTALL)/lib -ldshalcli -lds -L$(UTILS_PATH)/ -liarmUtils 
 
 ifeq ($(PLATFORM_SOC),L2HalMock)
-export LDFLAGS += -L$(DS_PATH)/install/lib -L$(IARM_PATH)/install -L$(UTILS_PATH)/ -liarmUtils -ldshalsrv -lds-hal -lds -ldshalcli -lIARMBus -lglib-2.0 -lsafec-3.5
+export LDFLAGS += -L$(DS_PATH)/install/lib -L$(IARM_PATH)/install -L$(UTILS_PATH)/ -liarmUtils -ldshalsrv -lds-hal -lds -ldshalcli -lIARMBus -lglib-2.0 -lsafec
 endif
 
 INTEL_SHLIBS=   \
