diff --git a/rpc/srv/dsDisplay.c b/rpc/srv/dsDisplay.c
index f7d5395..41f0477 100644
--- a/rpc/srv/dsDisplay.c
+++ b/rpc/srv/dsDisplay.c
@@ -72,7 +72,7 @@ IARM_Result_t _dsSetAllmEnabled(void *arg);
 IARM_Result_t _dsSetAVIContentType(void *arg);
 IARM_Result_t _dsSetAVIScanInformation(void *arg);
 IARM_Result_t _dsDisplayTerm(void *arg);
-void _dsDisplayEventCallback(intptr_t handle, dsDisplayEvent_t event, void *eventData);
+void _dsDisplayEventCallback(int handle, dsDisplayEvent_t event, void *eventData);
 static void  filterEDIDResolution(intptr_t Shandle, dsDisplayEDID_t *edid);
 static void  dumpEDIDInformation( dsDisplayEDID_t *edid);
 static dsVideoPortType_t _GetDisplayPortType(intptr_t handle);
@@ -506,7 +506,7 @@ IARM_Result_t _dsDisplayTerm(void *arg)
 
 
 /*HDMI Call back */
-void _dsDisplayEventCallback(intptr_t handle, dsDisplayEvent_t event, void *eventData)
+void _dsDisplayEventCallback(int handle, dsDisplayEvent_t event, void *eventData)
 {
 	IARM_Bus_DSMgr_EventData_t _eventData;
     IARM_Bus_DSMgr_EventId_t _eventId;
