diff --git a/rpc/include/dsserverlogger.h b/rpc/include/dsserverlogger.h
index b2df394..e79d02b 100644
--- a/rpc/include/dsserverlogger.h
+++ b/rpc/include/dsserverlogger.h
@@ -33,7 +33,7 @@
 #include "dsserverregisterlog.h"
 
 int ds_server_log(int priority,const char *format, ...);
-
+void dsServer_Rdklogger_Init();
 #if (defined(DSMGR_LOGGER_ENABLED))
 #include "rdk_debug.h"
 
diff --git a/rpc/srv/dsDisplay.c b/rpc/srv/dsDisplay.c
index f7d5395..3592886 100644
--- a/rpc/srv/dsDisplay.c
+++ b/rpc/srv/dsDisplay.c
@@ -72,7 +72,7 @@ IARM_Result_t _dsSetAllmEnabled(void *arg);
 IARM_Result_t _dsSetAVIContentType(void *arg);
 IARM_Result_t _dsSetAVIScanInformation(void *arg);
 IARM_Result_t _dsDisplayTerm(void *arg);
-void _dsDisplayEventCallback(intptr_t handle, dsDisplayEvent_t event, void *eventData);
+void _dsDisplayEventCallback(int handle, dsDisplayEvent_t event, void *eventData);
 static void  filterEDIDResolution(intptr_t Shandle, dsDisplayEDID_t *edid);
 static void  dumpEDIDInformation( dsDisplayEDID_t *edid);
 static dsVideoPortType_t _GetDisplayPortType(intptr_t handle);
